touchdown/ssh/client.py

```diff
@@ -52,16 +72,20 @@ def _run(self, transport, command):
             exit_status_ready = channel.exit_status_ready()
             while not exit_status_ready:
                 while channel.recv_ready():
-                    print(channel.recv(1024))
+                    buf = self._maybe_decode(channel.recv, 1024, input_encoding)
+                    print(buf, file=stdout)
```

Comment:
This breaks 2.7. I only used print here out of laziness - it should really go to the Plan() class that created this client and use it's .echo() method (which uses click under the hood). Thats what adds the [resource 'resource name'] stuff to the log line.